### ISSUE TITLE
Cannot run simplePingExample.py after pip install

### ISSUE THREAD

**Issue by jaxxzer** [bluerobotics/ping-python#35]
```
jack@jack-desktop:~$ python -m brping/ping1d
usage: ping1d.pyc [-h] --device DEVICE [--baudrate BAUDRATE]
ping1d.pyc: error: argument --device is required
jack@jack-desktop:~$ python -m simplePingExample.py
/usr/bin/python: No module named simplePingExample
jack@jack-desktop:~$ python simplePingExample.py
python: can't open file 'simplePingExample.py': [Errno 2] No such file or directory
```

**Comment by adrianrs79** [comment]
Hi Jaxxzer

I have a similar issue. In fact I got the following error
Traceback (most recent call last):
  File "simplePingExample.py", line 5, in <module>
    from brping import Ping1D
  File "build/bdist.linux-x86_64/egg/brping/__init__.py", line 2, in <module>
ImportError: No module named definitions

I guess is related to the path of the ping-sonar library, which is not detected, but I included it in the path and in the bash profile, any idea? 

Thanks in advance!
